examples/api/lib/widgets/tween_animation_builder/tween_animation_builder.0.dart

```diff
@@ -25,28 +29,35 @@ class TweenAnimationBuilderExampleApp extends StatelessWidget {
 }
 
 class TweenAnimationBuilderExample extends StatefulWidget {
-  const TweenAnimationBuilderExample({super.key});
+  const TweenAnimationBuilderExample({
+    this.duration = TweenAnimationBuilderExampleApp.duration,
```

Comment:
I think you can just hard code `const Duration(seconds: 1)` here as the default.  Having it be a separate static reference doesn't add anything to the example, and people will replicate it, which isn't necessarily desirable.  Instead of referencing the static in the test, just hardcode it there too.